nav2_util/include/nav2_util/lifecycle_utils.hpp

```diff
@@ -77,6 +77,33 @@ void reset_lifecycle_nodes(
   reset_lifecycle_nodes(split(nodes, ':'), service_call_timeout, retries);
 }
 
+/// Transition the given lifecycle nodes to the ErrorProcessing state in order
+/** At this time, service calls frequently hang for unknown reasons. The only
+ *  way to combat that is to timeout the service call and retry it. To use this
```

Comment:
I'm really curious, is this timeout issue still an issue? That was circa Bouncy when that message was added to this tool, we've come a long way since then.

Reply:
I actually have no idea, I added it as well for consistency with the other tools. I can take a look at it and see if this was fixed somewhere. Or if you know how to test it?

Reply:
Eh, lets just leave it. Its not hurting anyone. But I would be curious if in your unit tests you set the number of retries to 0 so that it fails and see if we ever catch it in your development / CI later. If it doesn't cause problems, we can remove it later.